guides/interview/preparatory-explanation.md

```diff
@@ -27,3 +27,37 @@ The below must be shared with the candidate prior to the interview itself:
   implemented in a language agreed with the candidate.
 - the tests will initially be failing and the aim of the assessment is to add or
   update the existing code such that the tests pass.
+
+## Tech Test Example
+
+The first question on our tech-test is the following (if taking the test in
+`python`):
+
+If you choose to research our company and read through our guides on the
+handbook we think that you should be allowed to see what kind of questions to
+expect on the test. There are 5 more questions beyond this one and they
+obviously get more challenging as they go along.
```

Comment:
Not to be _that guy_ but, is it obvious? I think maybe this reads better as:

```
There are 5 more questions beyond this one with the questions becoming more challenging as you progress through the test.
```

Or something like that?

Also, are there 5 more questions beyond this? I thought there were 5 questions total, which would make it 4 questions beyond this? :p